docs/contributor/02-01-installation.md

```diff
@@ -9,23 +117,23 @@ In such a setup, you don't need to install Keda Manager. It is installed and man
 
 When you enable the Keda module using your Kyma runtime Kyma custom resource (CR), the Lifecycle Manager downloads the bundled package of the Keda Manager and installs it. Additionally, it applies a sample Keda CR, which triggers Keda Manager to install the Keda module.
 
-![a](assets/keda-lm-overview.drawio.svg)
+![a](/docs/assets/keda-lm-overview.drawio.svg)
```

Comment:
The diagram is not formatted according to [our guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/09-diagrams.md).